### ISSUE TITLE
a bigger "show output" ?

### ISSUE THREAD

**Issue by chchia** [kapitainsky/RcloneBrowser#108]
as show in below picture, possible to have the show ouptup extended to all empty section in bottom? just to utilize the display output.

![image](https://user-images.githubusercontent.com/1291771/80275995-ea1d2e80-8717-11ea-88c5-96b9fd6fa8ea.png)

**Comment by kapitainsky** [comment]
and how it would work when you have not one but many transfers and let's say you expand two outputs and scroll the list?

**Comment by chchia** [comment]
i guess, perhap when you show MountA, it automatically hide MountB?

another scroll seems make very messy, scroll inside scroll? maybe not a good idea.

**Comment by kapitainsky** [comment]
yes it is definitely doable in some way but I don't see real benefit. Most people probably look at it sporadically or in case of troubleshooting.

**Comment by kapitainsky** [comment]
if somebody considers it important I am happy to accept PR

**Comment by kapitainsky** [comment]
so let's keep it on enhancements list - maybe somebody will do it

**Comment by chchia** [comment]
yeap thanks man, i am aware this only useful for debug purpose looking at the log with rclone -vv.

**Comment by kapitainsky** [comment]
I am working on next release with a lot of new features - if you would like to try latest pre-release link is in the first post in #93 thread
